Core/clim-basic/extended-streams/stream-input.lisp

```diff
@@ -358,62 +353,62 @@ keys read."))
 
 (defmethod stream-unread-char ((stream standard-extended-input-stream)
                                char)
   (with-encapsulating-stream (estream stream)
-    (stream-unread-gesture estream (unmap-char-for-read char))))
+    (stream-unread-gesture estream char)))
 
 (defmethod stream-peek-char ((stream standard-extended-input-stream))
   (with-encapsulating-stream (estream stream)
     (loop
-       with char and reason
-       do (setf (values char reason) (stream-read-gesture estream :peek-p t))
-       until (or (characterp char) (eq reason :eof))
-       do (stream-read-gesture estream) ; consume pointer gesture
-       finally (return (if (eq reason :eof)
-                           reason
-                           (char-for-read char))))))
+      with char and reason
+      do (setf (values char reason) (stream-read-gesture estream :peek-p t))
+      until (or (characterp char) (eq reason :eof))
+      do (stream-read-gesture estream) ; consume pointer gesture
+      finally (return (if (eq reason :eof)
+                          reason
+                          (char-for-read char))))))
 
 (defmethod stream-listen ((stream standard-extended-input-stream))
   (with-encapsulating-stream (estream stream)
     (loop
-       with char and reason
-         do (setf (values char reason) (stream-read-gesture estream
-                                                            :timeout 0
-                                                            :peek-p t))
-         until (or (characterp char) (eq reason :eof) (eq reason :timeout))
-         do (stream-read-gesture estream) ; consume pointer gesture
-         finally (return (characterp char)))))
+      with char and reason
+      do (setf (values char reason) (stream-read-gesture estream
+                                                         :timeout 0
+                                                         :peek-p t))
+      until (or (characterp char) (eq reason :eof) (eq reason :timeout))
+      do (stream-read-gesture estream) ; consume pointer gesture
+      finally (return (characterp char)))))
 
 (defmethod stream-clear-input ((stream standard-extended-input-stream))
   (with-encapsulating-stream (estream stream)
     (loop
-       with char and reason
-         do (setf (values char reason) (stream-read-gesture estream
-                                                            :timeout 0
-                                                            :peek-p t))
-         until (or (eq reason :eof) (eq reason :timeout))
-         do (stream-read-gesture estream) ; consume pointer gesture
-         ))
+      with char and reason
+      do (setf (values char reason) (stream-read-gesture estream
+                                                         :timeout 0
+                                                         :peek-p t))
+      until (or (eq reason :eof) (eq reason :timeout))
+      do (stream-read-gesture estream) ; consume pointer gesture
+      ))
```

Comment:
Move closing parentheses to previous line.

Reply:
this change was due to whitespace cleanup (indentation).